### PR TITLE
Fixed reamdme and sample app config link.

### PR DESCRIPTION
**Pull request by ronmonetaMicro**
### Added
* Support for user-based MI in Sample app
### Changed
* README and oneclick_instruction_box to match GO SDK
### Fixed
* Config JSON uri in the Sample app

### CODE REVIEW

File: quickstart/README.md [hunk 1]
```diff
@@ -10,8 +10,11 @@ You can use it as a baseline to write your own first kusto client application, a
 
 ### Prerequisites
 
-1. Set up Java version 8 or higher on your machine. For instructions, consult a Java environment setup tutorial, like [this](https://www.tutorialspoint.com/java/java_environment_setup.htm).
-1. Set up [Apache Maven](https://maven.apache.org/install.html), which is the most popular Java dependency management tool.
+- Go, version 1.19 or higher
```
Comment (yogilad) [inline quickstart/README.md:13]:
```suggestion
- [Go](https://go.dev/) version 1.19 or higher 
```

File: quickstart/oneclick_instruction_box.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
 ### Prerequisites
 
-1. Set up Java version 8 or higher on your machine. For instructions, consult a Java environment setup tutorial, like [this](https://www.tutorialspoint.com/java/java_environment_setup.htm).
-1. Set up [Apache Maven](https://maven.apache.org/install.html), which is the most popular Java dependency management tool.
+- Go, version 1.19 or higher
```
Comment (yogilad) [inline quickstart/oneclick_instruction_box.md:3]:
```suggestion
- [Go](https://go.dev/) version 1.19 or higher ```

File: quickstart/README.md [hunk 3]
```diff
@@ -10,8 +10,11 @@ You can use it as a baseline to write your own first kusto client application, a
 
 ### Prerequisites
 
-1. Set up Java version 8 or higher on your machine. For instructions, consult a Java environment setup tutorial, like [this](https://www.tutorialspoint.com/java/java_environment_setup.htm).
-1. Set up [Apache Maven](https://maven.apache.org/install.html), which is the most popular Java dependency management tool.
+- [Go](https://go.dev/) version 1.19 or higher 
+- An [Azure subscription](https://azure.microsoft.com/free/)
+- An [Azure Data Explorer Cluster](https://learn.microsoft.com/en-us/azure/data-explorer/).
+- An Azure Data Explorer Database. You can create a Database in your Azure Data Explorer Cluster using the [Azure Portal](https://learn.microsoft.com/en-us/azure/data-explorer/create-cluster-database-portal).
```
Comment (yogilad) [inline quickstart/README.md:16]: Or a free cluster.
Actually, a free cluster or this...

File: quickstart/README.md [hunk 4]
```diff
@@ -30,8 +33,8 @@ You can use it as a baseline to write your own first kusto client application, a
 ### Run the app
 
 1. Open a command line window and navigate to the folder where you extracted the app.
-1. Run `mvn clean install` to compile the source code into a binary.
-1. Run the binary using `java -jar target\kusto-quickstart-[version]-jar-with-dependencies.jar`.
+1. Run `go build` to compile the source code and files into a binary.
+1. Run the binary using `.\quickstart.exe` or whichever name you provided.
```
Comment (yogilad) [inline quickstart/README.md:37]: I assume the user needs to edit the config file manually (this is not the snippet).

File: quickstart/oneclick_instruction_box.md [hunk 5]
```diff
@@ -1,15 +1,17 @@
 ### Prerequisites
 
-1. Set up Java version 8 or higher on your machine. For instructions, consult a Java environment setup tutorial, like [this](https://www.tutorialspoint.com/java/java_environment_setup.htm).
-1. Set up [Apache Maven](https://maven.apache.org/install.html), which is the most popular Java dependency management tool.
+- [Go](https://go.dev/) version 1.19 or higher ```
+- An [Azure subscription](https://azure.microsoft.com/free/)
```
Comment (yogilad) [inline quickstart/oneclick_instruction_box.md:4]: Same (free cluster)

File: quickstart/oneclick_instruction_box.md [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
+- An [Azure Data Explorer Cluster](https://learn.microsoft.com/en-us/azure/data-explorer/).
+- An Azure Data Explorer Database. You can create a Database in your Azure Data Explorer Cluster using the [Azure Portal](https://learn.microsoft.com/en-us/azure/data-explorer/create-cluster-database-portal).
 
 ### Instructions
 
-1. Download the **DOWNLOAD_LINK** as a ZIP file.
-1. Extract the app source code.
+1. Download the app files from this GitHub repo.
+1. Modify the `kusto_sample_config.json` file, changing `KustoUri`, `IngestUri` and `DatabaseName` appropriately for your cluster.
```
Comment (yogilad) [inline quickstart/oneclick_instruction_box.md:11]: This is inveresed.
These instructions should appear on README.md and not on the one click instructions where the file is auto generated

File: quickstart/README.md (hunk 4 above)
Comment (ronmonetaMicro) [reply quickstart/README.md:37]: Not sure i understand what you mean

File: quickstart/oneclick_instruction_box.md (hunk 6 above)
Comment (ronmonetaMicro) [reply quickstart/oneclick_instruction_box.md:11]: They appear as well on the README. or did you mean something else?

**Pull request merged**